src/main/cpp/include/genomicsdb/genomicsdb_iterators.h

```diff
@@ -340,7 +341,6 @@ class SingleCellTileDBIterator {
   //Flag that specifies if this is the first time reading from TileDB
   bool m_first_read_from_TileDB;
   unsigned m_END_query_idx;
-  const VariantArraySchema* m_variant_array_schema;
```

Comment:
The GVCF iterator gets dim_domains from this object - we can either store the dim domains or just keep this object.

Reply:
We can keep this object. Thanks.